lib/models/movie.js

```diff
@@ -7,7 +7,7 @@ module.exports = Bookshelf.Model.extend({
   serialize: function () {
     return {
       id: this.get('id'),
-      title: this.get('title'),
+      name: this.get('name'),
```

Comment:
this should be `title: this.get('name')` since we dont want to break the customer implementation